_integration-schemas/klaviyo/bounce.md

```diff
@@ -0,0 +1,60 @@
+---
+tap: "klaviyo"
+version: "1.0"
+
+name: "bounce"
+doc-link: ""
+singer-schema: "https://github.com/singer-io/tap-klaviyo/blob/master/tap_klaviyo/schemas/bounce.json"
+description: ""
+
+replication-method: "Key-based Incremental"
+
+replication key: "state"
+
+api-method:
+    name: ""
+    doc-link: "https://www.klaviyo.com/docs/api/metrics"
+
+attributes:
+  - name: "id"
+    type: "string"
+    primary-key: true
+    description: ""
+    foreign-key-id: "bounce-id"
```

Comment:
Let's comment this out for now, since it doesn't point to any other tables:

```suggestion
    # foreign-key-id: "bounce-id"
```

Do this for all the other tables as well.